java/core/src/main/java/co/worklytics/psoxy/storage/impl/ColumnarBulkDataSanitizerImpl.java

```diff
@@ -293,6 +293,7 @@ public void sanitize(@NonNull Reader reader,
 
         CSVFormat csvFormat = CSVFormat.Builder.create()
             .setHeader(columnNamesForOutputFile.toArray(new String[0]))
+            .setRecordSeparator(records.getFirstEndOfLine())
```

Comment:
technically, a breaking change; previously separated records with `\r\n` regardless of what was in the input file.